collections/_projects/outdoorcase.md

```diff
@@ -0,0 +1,47 @@
+---
+name: "Outdoor cases for common devices"
```

Comment:
"common devices" might be too geneal. How about "wireless routers".
Or "Open Source outdoor cases for off the shelf wireless routers" - but maybe that is too long. :-)